project2/kernel.c

```diff
@@ -204,6 +207,7 @@ void Kernel_Task_Create() {
                 Process[x].wcet = request_info->wcet;
                 Process[x].ttns = sys_clock + request_info->offset;
                 Process[x].ticks_remaining = Process[x].wcet;
+                Process[x].clockState = clock_state;
```

Comment:
Good, this means that even if a process is created waaaaay late into the program's execution, the descriptor's clock state will be correct  👍